package.json

```diff
@@ -45,6 +45,7 @@
     "webpack-inline-svg-plugin": "^1.1.4"
   },
   "dependencies": {
+    "mermaid": "^9.4.0",
```

Comment:
Latest version 10.0.0 has issues with Webpack: https://github.com/mermaid-js/mermaid/issues/4094

Reply:
That issue was for v9.4.0. 
Can you share what issue you're having for 10.0.0?

Reply:
My bad, I might have overlooked this.
With 10.0.0, I'm getting:
```
ERROR in ./node_modules/mermaid/dist/mermaid.core.mjs 17:2-5
Can't import the named export 'l' from non EcmaScript module (only default export is available)
 @ ./src/app/services/code_service.js
 @ ./src/app/main/index.js
 @ ./src/app/index.run.js
```
This might be an issue in how webpack is configured, my knowledge is limited in this area. But simply changing the version to 9.4.0 does the trick :thinking:

Reply:
I had another look at this, mermaid 10 requires newer versions of webpack and friends, so I propose to keep this as an improvement for later